src/Endpoints.ts

```diff
@@ -42,6 +44,7 @@ export const Endpoints = {
       ancestors: <T extends ContentResponseElement>(id: string | number, options?: CDNContentAncestorsOptions) => new Endpoint<T[]>(EndpointSource.CDN, '/content/{id}/ancestors', {id}, 'get', options),
       descendants: <T extends ContentResponseElement>(id: string | number, options?: CDNContentDescendantsOptions) => new Endpoint<T[]>(EndpointSource.CDN, '/content/{id}/descendants', {id}, 'get', options),
       byContentType: <T extends ContentResponseElement>(contentType: string, options?: CDNContentByContentTypeOptions) => new Endpoint<PagedResponse<T>>(EndpointSource.CDN, '/content/type?contentType={contentType}',{contentType}, 'get', options),
+      filter: <T extends ContentResponseElement>(body: object, options?: CDNContentFilterOptions) => new Endpoint<PagedResponse<T>>(EndpointSource.CDN, '/content/filter', {}, 'post', options),
```

Comment:
We should make `body` a typed object since we know the structure.

It should allow the following structue:

```json
{
  "contentTypeAlias": "string?",
  "properties": [{
    "alias": "string",
    "value": "string",
    "match": "CONTAINS|LIKE"
  }]
}
```

As an example you could look at the form types (which have both array types and enums), it can be found [here](https://github.com/umbraco/Umbraco.Headless.Client.NodeJs/blob/master/src/Responses/Form.ts)